### DISCUSSION TITLE
More flexible bias-specific grid parameters

### DISCUSSION THREAD

**Discussion by jhenin** [Colvars/colvars#656] [Ideas]
I think histogramGrid is useful, and I would like to make it easier to use, and adopt it for ABF (and potentially metaD as well). A plan for that:

1. change the `f_cv_grid` feature - potentially remove it entirely, as the grid is not really a feature of the cv anymore. In particular, the requirement for boundaries at the colvar level are moot if grid parameters are given elsewhere
2. rename `histogramGrid` to `grid` so the same keyword can be used within different biases and documented once
3. Implement it in ABF

**Comment by giacomofiorin** [comment]
All are okay with me.

**Comment by jhenin** [comment]
Addressed by #710
